src/presentation/styles/colors.pcss

```diff
@@ -0,0 +1,7 @@
+:global(.light) {
+  --background: #fff;
+}
```

Comment:
When document won't have a `.light` or `.dark` class, there will be no variable values specified? Let's make `dark` values be specified by default